be/src/common/daemon.cpp

```diff
@@ -276,14 +276,14 @@ void Daemon::memory_gc_thread() {
     }
 }
 
-void Daemon::load_channel_tracker_refresh_thread() {
+void Daemon::memtable_flush_mgr_tracker_refresh_thread() {
     // Refresh the memory statistics of the load channel tracker more frequently,
     // which helps to accurately control the memory of LoadChannelMgr.
-    while (!_stop_background_threads_latch.wait_for(
-                   std::chrono::milliseconds(config::load_channel_memory_refresh_sleep_time_ms)) &&
```

Comment:
Backward compatibility is broken by just deleting this config.
`load_channel_memory_refresh_sleep_time_ms` should act as an alias of `memtable_flush_mgr_memory_refresh_sleep_time_ms`.

Reply:
Agree. But how to achive 'alias' semantic?